<launch>
  <node name="conductor" pkg="pipebot" type="conductor.py" output="screen" />
  <node name="DCMotor_subscriber" pkg="pipebot" type="DCMotor_subscriber.py" output="screen" />
  <node name="neato_laser_publisher" pkg="xv_11_laser_driver" type="neato_laser_publisher" output="screen" />
</launch>
